minard/pmtnoisedb.py

```diff
@@ -0,0 +1,27 @@
+from .db import engine_nl
+
+# turn a sql result into a list of dicts
+def dictify(rows):
+    keys = rows.keys()
+    rv = []
+    for row in rows:
+        d = {}
+        for k in keys:
+            d[k] = row[k]
+        rv.append(d)
+    return rv
+
+def runs_after_run(run, maxrun='+inf'):
+    conn = engine_nl.connect()
+    cmd = 'SELECT * FROM pmtnoise WHERE %d < run_number' % run
+    if maxrun < '+inf':
```

Comment:
Is this meant to be a string comparison? If maxrun is an integer you are probably best off having the default value be `None` and then just using `if maxrun:`.